pkg/api/api.go

```diff
@@ -91,8 +93,14 @@ func NewAPI(sigChan chan os.Signal, resetNamespaceFn func(namespaces *corev1.Nam
 
 	// GETs
 	r.Methods("GET").Path("/ready").HandlerFunc(h.isReadyHandler)
+
+	// monitoring
 	r.Methods("GET").Path("/metrics").Handler(promhttp.Handler())
 
+	// this is the cleanest way I found to register all pprof routes,
```

Comment:
I'd just change this too be less personal
`// this is the cleanest way I found to register all pprof routes,` -> `Known issue with Mux and the registering of pprof`